assets/js/modules/analytics/index.js

```diff
@@ -118,11 +119,11 @@ export const registerWidgets = ( widgets ) => {
 		'analyticsPopularPages',
 		{
 			Component: DashboardPopularPagesWidget,
-			width: widgets.WIDGET_WIDTHS.HALF,
+			width: [ widgets.WIDGET_WIDTHS.HALF, widgets.WIDGET_WIDTHS.FULL ],
```

Comment:
@eugene-manuilov I was wondering: will this mean that this widget could start appearing full width on the current dashboard and cause some regressions? 
If so, maybe we should use the feature flag to conditionally apply them?
Let me know what you think.

Reply:
Technically, it could appear full width on the current dashboard, but it will happen only if there are no other widgets fit into that row. So, I don't think we need to use the feature flag here.